packages/shared/src/lib/flow-run/execution/flow-execution.ts

```diff
@@ -0,0 +1,76 @@
+
+import { Static, Type } from '@sinclair/typebox'
+import { ExecutionState } from './execution-output'
+
+export enum FlowExecutionStatus {
```

Comment:
Since we are renaming, why not rename to `FlowRunStatus`? I think this will make it more clear instead of the mix between `run` and `execution` which is confusing.